modules/dhcp_isc/isc_state_changes_observer.rb

```diff
@@ -166,9 +166,10 @@ def update_subnet_service_with_dhcp_records(records)
               if dupe = service.find_host_by_mac(record.subnet_address, record.mac)
                 service.delete_host(dupe)
               end
-              if dupe = service.find_host_by_ip(record.subnet_address, record.ip)
-                service.delete_host(dupe)
-              end
+              #TODO: What is the aim of this?
```

Comment:
@witlessbird : I think, this is a very bad approach. When you have two records with the same ip address, conflict detection breaks because the first record is deleted and the second is never saved so smart-proxy doesn't know anything about either of them.
Is it ok if I remove this completely or am I missing something more?